server/templates/textbook.pug

```diff
@@ -14,6 +14,8 @@ html(lang=course.locale dir=dir)
       q-the-menu
 
     main.body
+      q-one-xp-banner
+
```

Comment:
for courses with multiple versions, we should get clarification if this is how it should look like (multiple spaced out banners)

<img width="1557" alt="image" src="https://github.com/Qiskit/platypus/assets/13156555/887a6b76-046c-4f91-ae72-458765a524d7">